pkgs/servers/nominatim/default.nix

```diff
@@ -0,0 +1,69 @@
+{
+  stdenv, lib, fetchFromGitHub, fetchurl,
+  clang-tools, cmake, bzip2, zlib, expat, boost, git, pandoc, gzip,
+  postgresql_12,
+  python3, python3Packages, php
```

Comment:
```suggestion
{ stdenv, lib, fetchFromGitHub, fetchurl
, clang-tools, cmake, bzip2, zlib, expat, boost, git, pandoc, gzip
, postgresql_12
, python3, python3Packages, php
```

Reply:
Applied in d7e9eb35e20efc5a56a564355e35ac4dbaf5f6d1